lib/fetch.js

```diff
@@ -42,7 +30,11 @@ export function sanitizeEventAndNews(item, type) {
     strapline: item.data[`${type}.strapline`] ?
       item.data[`${type}.strapline`].value : null,
     datetime: item.data[`${type}.timestamp`] ?
-      mapDate(item.data[`${type}.timestamp`].value) : null,
+      item.data[`${type}.timestamp`].value : null,
```

Comment:
@jameshopkins that `${type}.strapline.value` somehow doesn't work. It can only go up to the `${type}.strapline` level, which is not enough.